linalgo/annotate/models.py

```diff
@@ -407,6 +409,23 @@ def __init__(
         self.annotator = Annotator(annotator)
         self.task = Task(task)
         self.reviewee = Annotator(reviewee)
-    
+
     def __repr__(self) -> str:
-        return f'Schedule::{self.type}::{self.status}'
\ No newline at end of file
+        return f'Schedule::{self.type}::{self.status}'
+
+
+@dataclass
+class Body:
+    text: str = "N/A"
+    context: str = "N/A"
+    proba: float = -1.0
+    process: str = "N/A"
+
+
+class BodyFactory:
+    @staticmethod
+    def from_str(s):
+        data = ast.literal_eval(s)
+        if "body" in data and isinstance(data["body"], str):
```

Comment:
Please remove this line, too. Why would `body` also contain a body attribute?

Only
```
class BodyFactory:
    @staticmethod
    def from_str(s):
        data = ast.literal_eval(s)
```
should be enough. 

Also, we need to double check safety here. `ast.literal_eval(s)` is very flexible and could lead to unexpected behavior depending on the value provided by users (DDOS, etc).